templates/apiary-bucket-policy.json

```diff
@@ -12,6 +12,7 @@
             "Action": [
                 "s3:GetBucketLocation",
                 "s3:GetObject",
+                "s3:GetBucketAcl",
```

Comment:
If we are going to allow this for `customer_principals`, seems like we should allow it further down for `producer_iamroles` too.

Reply:
I'm ok with that. I couldn't decide since the specific tool we are doing this for isn't planned to be a producer, but I will add that.

Reply:
I added that now, please again @barnharts4.

Reply:
can we also add `s3:GetObjectAcl` ?